src/components/tag-input.js

```diff
@@ -39,12 +39,11 @@ export default class TagInput extends LitElement {
     const clipboardData = e.clipboardData || window.clipboardData;
     const pastedData = clipboardData.getData('Text');
     const pastedArray = pastedData ? pastedData.split(',').filter((v) => v.trim() !== '') : '';
-    
-    if(pastedArray) {
+
```

Comment:
eslint was complaining with errors at [0] while performing `yarn serve`. patched the whitespaces up to silence it.

[0]
```
WARNING in ./src/components/tag-input.js
Module Warning (from ./node_modules/eslint-loader/dist/cjs.js):

/home/actinium15/repo/mrin9/RapiDoc/src/components/tag-input.js
  42:1  error  Trailing spaces not allowed                                                   no-trailing-spaces
  43:5  error  Expected space(s) after "if"                                                  keyword-spacing
  46:7  error  Closing curly brace does not appear on the same line as the subsequent block  brace-style

✖ 3 problems (3 errors, 0 warnings)
  3 errors and 0 warnings potentially fixable with the `--fix` option.
```